TrafficReplayer/src/org/opensearch/migrations/replay/RequestResponseResponseTriple.java

```diff
@@ -0,0 +1,84 @@
+package org.opensearch.migrations.replay;
+
+import com.google.common.primitives.Bytes;
+import org.json.HTTP;
+import org.json.JSONObject;
+import org.opensearch.migrations.replay.TrafficReplayer.RequestResponsePacketPair;
+
+import java.io.BufferedOutputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.time.Duration;
+import java.util.List;
+
+public class RequestResponseResponseTriple<write> {
+    private RequestResponsePacketPair sourcePair;
+    private List<byte[]> shadowResponseData;
+    Duration shadowResponseDuration;
+
+    public RequestResponseResponseTriple(RequestResponsePacketPair sourcePair,
+                                         List<byte[]> shadowResponseData,
+                                         Duration shadowResponseDuration) {
```

Comment:
For some reason, calling this duration makes me think that this is the time that the response was going on for... but I think that you mean 'latency'.  I'd rename, though I realize that I'm being overly sensitive.